crops.csv

```diff
@@ -0,0 +1,372 @@
+geography,crops,total
```

Comment:
Just for my own understanding, if we've generated the SQL from CSV file, why do we still need to keep the CSV file in repo?